plugins/basic/nexus-repository-maven/src/main/java/org/sonatype/nexus/repository/maven/internal/maven2/Maven2GroupFacet.java

```diff
@@ -68,10 +73,35 @@
 public class Maven2GroupFacet
     extends GroupFacetImpl
 {
+  @VisibleForTesting
+  static final String CONFIG_KEY = "maven2group";
+
+  @VisibleForTesting
+  static class Config
+  {
+    @NotNull
+    public int contentMaxAge = 1440;
+
+    @NotNull
+    public String cacheToken = CacheController.newCacheToken();
+
+    @Override
+    public String toString() {
+      return getClass().getSimpleName() + "{" +
+          "contentMaxAge=" + contentMaxAge +
+          ", cacheToken=" + cacheToken +
+          '}';
+    }
+  }
+
   private final Maven2MetadataMerger metadataMerger;
 
+  private Config config;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Perhaps 'config' could be replaced by a local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=pmd%3ASingularField)